packages/graphql/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   graphql_parser: ^1.1.3
   rxdart: ^0.22.0
 dev_dependencies:
-  perfectionist: ^2.0.0
+  pedantic: "<=1.7.99"
```

Comment:
Why not latest?

Reply:
the latest is conflict with `flutter_test` package, which requires exactly `1.5.0` (i tried and the dependency resolver failed)